com.github.maliit.keyboard.metainfo.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="inputmethod">
+  <id>com.github.maliit.keyboard</id>
+  <name>Maliit Keyboard</name>
+  <summary>A virtual keyboard for touch devices</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-3.0-only</project_license>
+  <developer_name>Maliit Team</developer_name>
+  <description>
+    <p>Based on Ubuntu Keyboard. Ubuntu Keyboard was a QML and C++ based Keyboard Plugin for Maliit, based on the Maliit Reference plugin, taking into account the special UI/UX requests of Ubuntu Phone.</p>
```

Comment:
I don't think a history lesson makes for a good description. (The Readme here has the same but it's not really end user-facing documentation.)

Reply:
Agreed, I just copied because I didn't want to impose a description for the project. I'm open to alternatives.

Reply:
It's not really imposing when you make a pull request. Jan can refuse at any time. ;) I'm currently having writer's block and can't come up with something either. If you care for some inspiration, have a look at https://en.wikipedia.org/wiki/Maliit

I basically wrote the entire article. Changes by others were very minor (probably not passing the threshold of originality). Feel free to use anything I wrote under CC0.

Reply:
Thanks @KAMiKAZOW, that was very helpful. :)

Reply:
I ended up being a bit more creative as this is for end-users rather than for developers. I hope it makes sense to you all.